README.md

```diff
@@ -1,261 +1,79 @@
-<h1 align="center">
-    <br>
-    Av1an
-    </br>
-</h1>
+# Av1an
 
-<h2 align="center">A cross-platform framework to streamline video encoding</h2>
+[![Discord server](https://discordapp.com/api/guilds/696849974230515794/embed.png)](https://discord.gg/Ar8MvJh)
+[![CI tests](https://github.com/master-of-zen/Av1an/actions/workflows/tests.yml/badge.svg)](https://github.com/master-of-zen/Av1an/actions/workflows/tests.yml)
+[![](https://img.shields.io/crates/v/av1an.svg)](https://crates.io/crates/av1an)
 
-![alt text](https://cdn.discordapp.com/attachments/804148977347330048/928879953825640458/av1an_preview.jpg)
+Av1an is a video encoding framework for modern encoders. It can increase your encoding efficiency by running multiple encoder processes in parallel. This can improve CPU usage and increases the speed of some AV1 encoders dramatically.
 
-<h4 align="center">
-<a href="https://discord.gg/Ar8MvJh"><img src="https://discordapp.com/api/guilds/696849974230515794/embed.png" alt="Discord server" /></a>
-<img src="https://github.com/master-of-zen/Av1an/workflows/tests/badge.svg">
-<a href="https://crates.io/crates/av1an"><img src="https://img.shields.io/crates/v/av1an.svg"></a>
+Av1an can also calculate a VMAF score for you to assess the encode quality, and it can even target a specific VMAF score when encoding.
 
-</h4>
-<h2 align="center">Easy, Fast, Efficient and Feature-Rich</h2>
+## Features
 
-### <center>An easy way to start using AV1, HEVC/H.265, AVC/H.264, VP9, and VP8 encoders.<br> AOM, RAV1E, SVT-AV1, VPX, x265, x264 are supported.</center>
+- Vastly improved encoding speed for some encoders
+- Cancel and resume encoding without loss of progress
+- [VapourSynth](http://www.vapoursynth.com) script support
+- [Target Quality](/docs/TargetQuality.md) mode, using [VMAF](https://github.com/Netflix/vmaf) to automatically set encoder options to achieve the wanted visual quality
+- Simple and clean console interface
+- Convenient Docker images available
+- Cross-platform application written in Rust
 
-Example with default parameters:
+## Supported encoders
 
-    av1an -i input
+At least one encoder is required to use Av1an. Install any of these that you wish to use.
```

Comment:
Might be good to clarify the standalone binaries for each encoder that are needed for av1an to be able to use them? For example, vpxenc is needed for libvpx, and it's possible to have libvpx installed without the vpxenc binary (and this does happen in practice for some package managers).

Reply:
Sure, I replaced the lib names with the encoder names (libvpx -> vpxenc) and put a note below the list.